resource_retriever/include/resource_retriever/exception.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright 2009, Willow Garage, Inc. All rights reserved.
```

Comment:
Do we want to update the copyright headers to 2025 and attribute OSRF?

Reply:
We can add an OSRF 2025 but we can't change it.